releasenotes/notes/fix_clusterchecks_secrets_scrubbing-4caa0177b3c33851.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Cluster Agent now scrubs secrets when generating flares and cluster check reports.
```

Comment:
This only affects the cluster-agent, not the node-agent. When that's the case, release notes go in `releasenotes-dca/notes`.

Also, maybe we should be more specific here. flares where working as expected. The fix is for the `agent clusterchecks` command.